src/style/component.scss

```diff
@@ -0,0 +1,151 @@
+.App {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  .circle-inner {
+    position: fixed;
+    width: 5px;
+    height: 5px;
+    border-radius: 2.5px;
+    background-color: black;
+    transition: 0.1s;
+    pointer-events: none;
+  }
+  .circle-outer {
+    position: fixed;
+    width: 20px;
+    height: 20px;
+    border-radius: 10px;
+    background-color: transparent;
+    border: 2px solid gray;
+    transform: translate(-7.5px, -7.5px);
+    transition: 0.2s;
+    pointer-events: none;
+  }
+  .App-header{
+    position: fixed;
+    display: flex;
+    justify-content: center;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: var(--height-nav);
+    padding-top: 10px;
+    &__container{
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 90%;
+      .nav{
+        display: flex;
+        align-items: center;
+        height: 40px;
+        position: relative;
+        ul{
+          list-style: none;
+          .nav-li{
+            display: inline-block;
+            width: 100px;
+            text-align: center;
+            a{
+              text-decoration: none;
+              font-size: 1.6rem;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .content {
+    margin-top: var(--height-nav);
+    width: 90%;
+    font-size: 1.4rem;
+    display: flex;
+    .author-container{
+      width: 700px;
+      height: calc(100vh - var(--height-nav) - 50px);
+    }
+    .main-content {
+      flex: 1;
+    }
+    .home{
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      h1{
+        font-size: 3.5rem;
+        height: 100px;
+         display: inline-block;
+         padding-bottom: 5%;
+         margin-bottom: 5%;
+         position: relative;
+         &::after{
+          content: "";
+          position: absolute;
+          top: 100%;
+          left: 0;
+          width: 30%;
+          height: 5px;
+          border-radius: 2.5px;
+          background-color: black;
+         }
+      }
+      .create-section{
+        display: flex;
+        align-items: center;
+        h2{
+          font-size: 1.6rem;
+        }
+        ul{
+          li{
+            width: 110px;
+            text-align: center;
+            display: inline-block;
+          }
+        }
+      }
+      &-btn{
+        margin-top: 50px;
+      }
+    }
+  }
+
+  footer{
+    position: fixed;
```

Comment:
let's have a classname for the footer, try to avoid using the default element name